src/style.css

```diff
@@ -0,0 +1,41 @@
+@font-face {
+  font-family: Avenir, Times, serif;
+  src: url(./fonts/Avenir.ttc),
+    url(./fonts/Avenir-Book.ttf),
+    url(./fonts/Avenir-Heavy.ttf);
+}
+
+.searchForm {
+  margin: 0 0 20px;
+}
+
+.search {
+  width: 100%;
+  padding: 16px 32px;
+  font-weight: 300;
+  text-indent: 8px;
+  background: white url(./images/Search.svg) 1em no-repeat;
+  border: 1px solid rgba(200, 200, 200, 0.75);
+  border-radius: 4px;
+  box-shadow: 0 1px 1px rgba(200, 200, 200, 0.5);
+}
+
+.searchMain {
+  font-style: avenir-heavy;
```

Comment:
you don't have this font, create valid font-faces to assign .ttf to what you write in stylesheets